<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellService</class>
 <widget class="QWidget" name="CellService">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CellService</string>
  </property>
  <widget class="QLabel" name="RGB_QLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>371</width>
     <height>331</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>-3</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Blue_QLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>380</y>
     <width>371</width>
     <height>331</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>-3</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Red_QLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>371</width>
     <height>331</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>-3</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Green_QLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>371</width>
     <height>331</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>-3</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
